packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -162,6 +162,72 @@ When the `smsRole` property is specified, the `smsRoleExternalId` may also be sp
 assume role policy should be configured to accept this value as the ExternalId. Learn more about [ExternalId
 here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
 
+User pools can be configured to enable MFA. It can either be turned off, set to optional or made required. Setting MFA
+to optional means that individual users can choose to enable it.
+Additionally, the MFA code can be sent either via SMS text message or via a time-based software token.
+See the [documentation on MFA](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-mfa.html) to
+learn more.
+
+The following code snippet marks MFA for the user pool as required. This means that all users are required to
+configure an MFA token and use it for sign in. It also allows for the users to use both SMS based MFA, as well,
+time-based one time password.
+
+```ts
+new UserPool(this, 'myuserpool', {
+  // ...
+  mfaEnforcement: MfaEnforcement.REQUIRED,
+  mfaToken: {
+    sms: true,
+    oneTimePassword: true,
+  },
+});
+```
+
+User pools can be configured with policies around a user's password. This includes the password length and the
+character sets that they must contain.
+
+Further to this, it can also be configured with the validity of the auto-generated temporary password. A temporary
+password is generated by the user pool either when an admin signs up a user or when a password reset is requested.
+The validity of this password dictates how long to give the user to use this password before expiring it.
+
+The following code snippet configures these properties -
+
+```ts
+new UserPool(this, 'myuserpool', {
+  // ...
+  passwordPolicy: {
+    minLength: 12,
+    requireLowercase: true,
+    requireUppercase: true,
+    requireDigits: true,
+    requireSymbols: true,
+    tempPasswordValidity: Duration.days(3),
+  },
+});
+```
+
+Note that, `tempPasswordValidity` can be specified only in days.
+
+### Emails
+
+Cognito sends emails to users in the user pool, when particular actions take place, such as welcome emails, invitation
+emails, password resets, etc. The address from which these emails are sent can be configured on the user pool.
+
+```ts
+new UserPool(this, 'myuserpool', {
+  // ...
+  emailTransmission: {
```

Comment:
```suggestion
  email: {
```

Reply:
This is particularly around settings for how emails are sent. Leaving this as email can be confused with other parts of the user pools such as verification emails, invitation emails, etc.

I can change this to `emailSettings` if that would be clearer?

Reply:
Maybe `emailOption`?